pkg/ipmi/blobs/blob_transfer.go

```diff
@@ -333,6 +339,7 @@ func (h *BlobHandler) BlobWrite(sid SessionID, offset uint32, data []uint8) erro
 //
 // Each blob defines its own commit behavior. Optional blob-specific commit data
 // can be provided with |data|.
+// This function does not poll for BMC_BLOB_STATE_COMMITTED.
 func (h *BlobHandler) BlobCommit(sid SessionID, data []uint8) error {
```

Comment:
could you this take an io.Reader instead of a slice. Further, no matter what you pass, i think covering it with a test is pretty easy.